pom.xml

```diff
@@ -95,7 +95,7 @@
         <jetty.version>9.4.3.v20170317</jetty.version>
         <lucene.version>4.10.4</lucene.version>
         <spring.version>4.2.4.RELEASE</spring.version>
-        <spring.security.version>4.0.3.RELEASE</spring.security.version>
+        <spring.security.version>4.2.3.RELEASE</spring.security.version>
```

Comment:
I'm not aware of the changes. Just realized we hadn't upgraded in awhile. I'm happy to back this out and create another JIRA to investigate further.